eXo/Sources/Controllers/ConnectToExoVC/ConnectToExoViewController.swift

```diff
@@ -118,7 +118,24 @@ extension ConnectToExoViewController:UITableViewDelegate,UITableViewDataSource{
         cell.setupDataWith(serveur:serveur)
         cell.deleteButton.tag = indexPath.row
         cell.deleteButton.addTarget(self, action: #selector(deleteButtonTapped(_ :)), for: .touchUpInside)
-        cell.badgeView.isHidden = true
+        // Handle Badge Notififcation appearance in case of session timeout
+        if let _quitTimestamp = defaults.value(forKey: "quitTimestamp") as? Double {
```

Comment:
This is not a valid check for session timeout. The best will be to use this URL to get the user status https://url.of.exo/portal/rest/state/status/${userName}
could you use this to check the user connectivity ?